fern/pages/docs/introduction/project-structure.mdx

```diff
@@ -0,0 +1,129 @@
+---
+title: Project Structure
+excerpt: An overview of the file and folder structure of a Fern Docs project
+---
+
+This page provides an overview of the file and folder structure of a Fern Docs project.
+
+## Top-level Folders 
+
+<CodeBlock title="fern/">
+```bash
+  fern
+  ├─ pages
+  ├─ assets
+  ├─ docs.yml
+  ├─ openapi
+  └─ fern.config.json
+```
+</CodeBlock>
+
+A Fern Docs project has the following top-level folders:
+
+- `pages`: Contains the Markdown (MDX) files that make up your documentation.
+- `assets`: Contains any images or videos used in your documentation.
+- `docs.yml`: The configuration file that defines the navigation, theme, and hosting details of your documentation.
+- `openapi`: Contains the OpenAPI specification file (if you have an API Reference section in your documentation).
+- `fern.config.json`: The configuration file specifying your organization name and CLI version.
+
+## Pages Folder
+
+The `pages` folder contains the Markdown (MDX) files that make up your documentation. Each MDX file represents a page in your documentation.
+
+<CodeBlock title="fern/pages">
+```bash
+  pages
+  ├─ introduction
+  │  ├─ quickstart.mdx
+  │  ├─ project-structure.mdx
+  │  └─ showcase.mdx
+  ├─ building-your-docs
+  │  ├─ navigation
+  │     ├─ sections.mdx
+  │     ├─ tabs.mdx
+  │     └─ versions.mdx
+  └─ └─ configuration.mdx
+
+```
+</CodeBlock>
+
+The `pages` folder is organized into subfolders based on the sections of your documentation. Each subfolder contains the MDX files for the pages in that section.
+
+## Assets Folder
+
+The `assets` folder contains any images or videos used in your documentation. You can reference these assets in your MDX files using relative paths.
+
+<CodeBlock title="fern/assets">
+```bash
+  assets
+  ├─ favicon.ico
+  ├─ product-screenshot.svg
+  ├─ demo-video.mp4
+  ├─ logo-dark-mode.png
+  └─ logo-light-mode.png
+```
+</CodeBlock>
+
+## Docs.yml File
+
+The `docs.yml` file is the configuration file that defines the navigation, theme, and hosting details of your documentation. You can customize the appearance and behavior of your documentation by editing this file.
+
+<CodeBlock title="fern/docs.yml of this website">
+```yml
+instances:
+  - url: fern.docs.buildwithfern.com/learn
+    custom-domain: buildwithfern.com/learn
+
+navigation:
+  - section: Introduction
+    layout:
+      - page: QuickStart
+        path: pages/introduction/quickstart.mdx
+      - page: Project Structure
+        path: pages/introduction/project-structure.mdx
+      - page: Showcase
+        path: pages/introduction/showcase.mdx
+
+navbar-links:
+  - type: filled
+    text: Book a demo
+    url: https://buildwithfern.com/contact
+
+logo:
+  light: ./images/logo-green.png
+  dark: ./images/logo-white.png
+
+colors:
+  accentPrimary:
+    dark: "#ADFF8C"
+    light: "#209d63"
+
+favicon: ./images/favicon.ico 
+
+title: Fern's Documentation
+```
+</CodeBlock>
+
+## OpenAPI Folder
```

Comment:
I think we should call this API Definitions and have an accordion that lists: 
- OpenAPI
- FernDefinition
- Multiple APIs